app/config/providers.php

```diff
@@ -101,6 +101,16 @@
         'beta' => false,
         'mock' => false,
     ],
+    'Etsy' => [
```

Comment:
```suggestion
    'etsy' => [
```

The Keys for the providers are expected to be lower cased 👀